mne/coreg.py

```diff
@@ -1223,3 +1225,528 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
                 F_mri_ras, 'ras', 'ras'),
             F_ras_mni, 'ras', 'mni_tal')
     _write_fs_xfm(fname_to, T_ras_mni['trans'], kind)
+
+
+class Coregistration(object):
+    """
+    Performs automated coregistration.
+
+    Parameters
+    ----------
+    info : instance of Info
+        The measurement info.
+    subjects_dir : directory
+        SUBJECTS_DIR.
+    subject : str
+        Subject, corresponding to a folder in SUBJECTS_DIR.
+    """
+
+    def __init__(self, info, subjects_dir, subject):
```

Comment:
```suggestion
    def __init__(self, info, subject, subjects_dir=None):
```
subject is commonly passed first and subjects_dir can be None in which case we find it from the os.environ
See how it's done elsewhere